defaults/main.yml

```diff
@@ -23,6 +23,9 @@ postgresql_service_enabled: true
 postgresql_cluster_name: "main"
 postgresql_cluster_reset: false
 
+# How strong should the SSL key be
```

Comment:
I deliberately put this here, and not down with `postgresql_ssl` because this is not a `postgresql.conf` parameter, but a part of the initial installation.